connectivity/check/policy.go

```diff
@@ -59,9 +67,17 @@ func (e ExitCode) Check(code uint8) bool {
 func (ct *ConnectivityTest) getCiliumPolicyRevisions(ctx context.Context) (map[Pod]int, error) {
 	revisions := make(map[Pod]int)
 	for _, cp := range ct.ciliumPods {
-		revision, err := getCiliumPolicyRevision(ctx, cp)
-		if err != nil {
-			return revisions, err
+		var revision int
+		var err error
+		for i := 1; i <= getPolicyRevisionRetries; i++ {
+			revision, err = getCiliumPolicyRevision(ctx, cp)
+			if err == spdystream.ErrTimeout {
```

Comment:
Does it even matter why this failed? I would argue we should just retry regardless.

Reply:
I'd rather be conservative at first given retries are usually a last resort. If we ever see it fail with something else, we can consider extending.

Reply:
Sure, but given that this is a read-only request that, if failed, breaks the entire test run, this feels like unnecessary coupling to a single error type. apiservers can blip on, say, dns resolution, and others may not talk SPDY (e.g. ones behind a http/1 L7 proxy).